src/short-urls/ShortUrlsList.js

```diff
@@ -50,9 +50,22 @@ const ShortUrlsList = (ShortUrlsRow) => class ShortUrlsList extends React.Compon
       return null;
     }
 
+    let icon = null;
+
+    switch (this.state.orderDir) {
+      case 'ASC':
+        icon = caretUpIcon;
+        break;
+      case 'DESC':
+        icon = caretDownIcon;
+        break;
+      default:
+        return null;
+    }
```

Comment:
I have never been a big fan of `switch` statements or using `let` if it is not absolutely unavoidable :sweat_smile:

In this case I think we can simplify it. What do you think about this other approach:

```js
if (!this.state.orderDir) {
  return null;
}

const icon = this.state.orderDir === 'ASC' ? caretUpIcon : caretDownIcon;

return <FontAwesomeIcon icon={icon} className="short-urls-list__header-icon" />;
```